explainaboard/explainaboard_main.py

```diff
@@ -40,22 +48,21 @@ def get_tasks(task: str, system_outputs: list[str]) -> list[str]:
             )
         return real_tasks
     else:
-        file_type = FileType.json
-        dummy_task = TaskType.text_classification
         for sys_output in system_outputs:
-            loader = get_custom_dataset_loader(
-                dummy_task,
-                sys_output,
-                sys_output,
-                dataset_file_type=file_type,
-                output_file_type=file_type,
-            )
-            if not loader.user_defined_metadata_configs:
+            task_or_die: TaskType | None = None
```

Comment:
The name of this variable (`task_or_die` ) is so interesting that If want to know how you are going to add a docstring for it :)

Reply:
Your wish is my command (in the next commit ;)).